src/Entity/Figure.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\FigureRepository")
+ */
+class Figure
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $dateCreation;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $dateLastModification;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\FiguresGroup", inversedBy="figures")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $figuresGroup;
+
+
+    public function __construct()
+    {
+        $this->dateCreation = new \DateTime();
```

Comment:
```suggestion
         $this->dateCreation = new \DateTimeImmutable();
```
Théoriquement la date ne doit jamais être changer donc.

Reply:
fait